src/selectors/app.js

```diff
@@ -338,3 +340,9 @@ export const getTimelineMarginLeft: Selector<number> = createSelector(
     }
   }
 );
+
+export const getL10nState: Selector<L10nState> = state => state.l10n;
+export const isL10nFetching: Selector<boolean> = state =>
+  getL10nState(state).fetching;
+export const getLocalizations: Selector<Localization> = state =>
```

Comment:
`getLocalizations` is plural but it returns `localization` which is singular. It would be better to make them consistent.